src/blocks/Slider/Slider.tsx

```diff
@@ -289,14 +307,20 @@ export const SliderBlock = (props: React.PropsWithChildren<SliderProps>) => {
     };
 
     const renderDot = (index: number) => {
+        const isVisible = isVisibleSlide(index);
         return (
             <li
                 key={index}
                 className={b('dot', {active: index === currentIndex})}
                 onClick={() => handleDotClick(index)}
-                aria-hidden={isVisibleSlide(index) ? true : undefined}
-                aria-label={`Slide ${getCurrentSlideNumber(index)} of ${barSlidesCount}`}
-            ></li>
+                role="button"
+                aria-hidden={isVisible ? true : undefined}
+                tabIndex={isVisible ? -1 : 0}
```

Comment:
It is preferred to implement the so called "Roving tabindex" https://www.tohuwabohu.io/2022/10/roving-tabindex/#menus-and-interactive-lists

It is much better, as we can focus only on 1 dot instead of all visible ones and change the active tab using arrow keys